doc/install-dependencies.md

```diff
@@ -53,7 +53,7 @@ sudo yum install yum-utils
 sudo yum-config-manager --add-repo https://openresty.org/package/centos/openresty.repo
 
 # install OpenResty and some compilation tools
-sudo yum install -y openresty curl git gcc luarocks lua-devel
+sudo yum install -y openresty curl git gcc luarocks lua-devel openresty-openssl-devel
```

Comment:
Look like we should also install luarocks from the source?
We can do it with utils/linux-install-luarocks.sh, but we need to move the `OPENSSL` variable configuration to this script.

Reply:
OK.

Reply:
@spacewander This is the document for installing dependencies.

Reply:
I think we could hint the users to install LuaRocks 3.x, and add the operations for OPENSSL variables here.

Reply:
Need to remove luarocks from this list.

Reply:
Fixed.